consumer/src/main/java/com/autostreams/kafka/ConsumerWorker.java

```diff
@@ -65,8 +60,8 @@ public void stop() {
     /**
      * Creates the relevant kafka consumer and subscribes it to specified topics.
      */
-    private void createConsumer() throws IOException {
-        Properties props = FileUtils.loadConfigFromFile("consumerconfig.properties");
+    private void createConsumer() {
+        Properties props = FileUtils.loadPropertiesFromFile("consumerconfig.properties");
 
         String host = System.getenv().getOrDefault("KAFKA_BROKER_URL",
             props.getProperty("kafka.url", "127.0.0.1")
```

Comment:
Should we do something similar here to what we have done with the producer? Might be out of scope for this commit